File: MonoTouch/PresentationGauge_MT/MainViewController.cs
using System;
using MonoTouch.UIKit;
using System.Drawing;
using MonoTouch.Foundation;
using MonoTouch.CoreAnimation;
using MonoTouch.CoreGraphics;

namespace mt_test1
{
	public class MainViewController : UIViewController
	{
		private GaugeView gaugeView;
		
		public override void LoadView ()
		{
			View = gaugeView = new GaugeView();
		}
		
		public override void ViewDidAppear(bool animated)
		{
			base.ViewDidAppear(animated);
			
			NSTimer timer = NSTimer.CreateRepeatingTimer(3.0, delegate { gaugeView.Value = DataService.GetScore(); });
			
			NSRunLoop.Current.AddTimer(timer,NSRunLoopMode.Default);
		}
	}
}

File: iPhone/PresentationGauge/PresentationGauge/MainViewController.m
#import "MainViewController.h"
#import "GaugeView.h"
#import "DataService.h"

@implementation MainViewController

- (void)loadView
{
    self.view = [[[GaugeView alloc] init] autorelease];
}

- (void)viewDidLoad
{
    [super viewDidLoad];
    
    NSTimer *timer = [NSTimer timerWithTimeInterval:3.0 target:self selector:@selector(timerFired) userInfo:nil repeats:YES];
                      
    [[NSRunLoop currentRunLoop] addTimer:timer forMode:NSRunLoopCommonModes];
}

- (void) timerFired
{
    DataService *dataService = [[DataService alloc] init];
    
    int score = [dataService getScore];
    
    [dataService release];
    
    [((GaugeView *)self.view) setScore:score];
}
                      
@end
